### DISCUSSION TITLE
Set_ Get Add @ itemComposition API

### DISCUSSION THREAD

**Discussion by osbScuffi** [runelite/runelite#18055] [Ideas]
Hi,
could u make get & set for **'Zoom2d' 'xoffset2d' 'yoffset2d'** on the **'ItemComposition'** API public?

Need to make use of `ItemComposition.SetZoom2d();`
Thank u!

**Comment by ldavid432** [comment]
Seconded, would love to make a plugin that messes with item sprites and without the offset and zoom it makes it hard to do much
